docs/task_kg_link_tail_prediction.md

```diff
@@ -173,22 +179,21 @@ pythonic interface provided by ExplainaBoard.
 ### Customized Bucket Order
 In some situation, users aim to specify the bucket order according to their needs. Following [code](https://github.com/neulab/ExplainaBoard/blob/8ccd1a71531bc3b9e2f9e539cb001353cc49ebca/explainaboard/tests/test_kg_link_tail_prediction.py#L60) gives an example.
 
-
+Below is an example of sorting alphabetically by bucket value:
```

Comment:
Overall comment: these code examples seem a bit verbose for a `.md` file. If they could be made a bit more concise that'd be nice, but it's not an urgent issue.